bubble_sort_by.rb

```diff
@@ -0,0 +1,28 @@
+def bubble_sort_by(array)
+	(array.length).times do |x|
+		(array.length - (x + 1)).times do |y|
+			if array[y + 1] != nil
```

Comment:
The ruby way to check for nil values is `.nil?` method. Use` if !array[arr + 1].nil?` instead